index.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const Bitmap = require(`${__dirname}/model/bitmap-constructor.js`);
+const ColorTransform = require(`${__dirname}/model/color-constructor.js`);
+const fileHelper = require(`${__dirname}/lib/bitmap-file-helper.js`);
+
+module.exports = exports = {};
+
+exports.invertBitmap = () => {
```

Comment:
Not taking in a path from anywhere on this means that you only have access to one bitmap file and can't test multiple. It would be good to keep this modular in case you want to read other bitmap files.